engine/src/main/java/org/terasology/physics/components/RigidBodyComponent.java

```diff
@@ -48,6 +48,9 @@
     @Replicate
     public float friction = 0.5f;
 
+    @Replicate
+    public float restitution = 0f;
```

Comment:
Wish list: simple javadoc on the two fields that spell out the difference :D

Friction is pretty easy, other one a bit less purely on the name (figuring bouncy)

Reply:
I also had to search the internets to understand the difference.  Ill find a good explanation.